CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## next
+- Updated versions for bugfixes + improved macOS support.
+- Update flutter_svg to 0.17.1 [Breaking Change]
```

Comment:
Is this really a breaking change? I mean, for people who uses Flame, there is no changes required right?

Reply:
Yes, this is correct. There is no change for using flame, I just wanted to show that I had to change a method in flutter svg while the other plugins were updated without any changes